docs/source/api/modules.rst

```diff
@@ -1,8 +1,7 @@
-API Documentation
-=================
+transformers4rec
```

Comment:
I know this is autogenerated - but we should have the previous 'API Documentation ' header here.